apps/api/src/status/test/controller.spec.ts

```diff
@@ -284,7 +284,7 @@ describe("Status", () => {
           },
           workers: {
             activated: 1,
-            fresh: 1,
+            fresh: 0,
```

Comment:
Instead of changing the behaviour, could you try to wait for one second to let api get ready workers?
Let me know if it doesn't work.